src/quadpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -149,15 +149,22 @@ def process(self, instance):
         make_playblast = instance.data["creator_attributes"].get("make_playblast", False)
         export_type = instance.data["creator_attributes"].get("export_type", "scene")
         apply_background = instance.data["creator_attributes"].get("apply_background", True)
+        apply_alpha_background = instance.data["creator_attributes"].get("apply_alpha_background", False)
         is_review = instance.data["family"] == "review"
 
+        if apply_background and apply_alpha_background:
+            raise ValueError("The options \"Disable BG\" and \"Apply BG Color\" are BOTH activated \n"
+                             "Please disable one of them to proceed")
+
         if is_review or make_playblast:
             result = self.render_review(
                 output_dir,
                 export_type,
                 mark_in,
                 mark_out,
-                scene_bg_color if apply_background else None,
+                scene_bg_color,
```

Comment:
Following my previous comment, this would merge these 3 variables with only a `scene_bg_color`